modules/web/js/ballerina/item-provider/initial-definitions.js

```diff
@@ -254,17 +254,17 @@ var createThrowStatementTool = {
 var createAbortStatementTool = {
     id: "Abort",
     name: "Abort",
-    iconSrc: require("./../../../images/tool-icons/throw.svg"),
-    cssClass: "icon fw fw-throw",
+    iconSrc: require("./../../../images/tool-icons/abort.svg"),
```

Comment:
This will be removed later on. As we are going to remove all the svg file imports. And use Font WSO2 icons.json in the graph and Font Icons in HTML. So its not really necessary.